docs/scripts/buildApi.ts

```diff
@@ -1309,8 +1304,8 @@ function run(argv: { componentDirectories?: string[]; grep?: string; outputDirec
       theme,
       workspaceRoot,
     })
-      .then((value) => {
-        return { status: 'fulfilled' as const, value };
```

Comment:
Is `void` anyway. This change and https://github.com/mui-org/material-ui/pull/24504/files#diff-94485904894207581ab0136f54456a2f976658c5983c4d0f776a00dd4e4c6bbeR1296 are drive-by refactorings to simplify the return value:
```diff
-Promise<{status: 'fulfilled', value: void} | {status: 'rejected', error: any}> | { status: string }
+Promise<{status: 'fulfilled'} | { status: 'rejected', error: any}>
```